frontend/src/static/js/components/webstatus-overview-cells.ts

```diff
@@ -66,14 +67,23 @@ export enum ColumnKey {
   AvailabilityEdge = 'availability_edge',
   AvailabilityFirefox = 'availability_firefox',
   AvailabilitySafari = 'availability_safari',
+  AvailabilityChromeAndroid = 'availability_chrome_android',
+  AvailabilityFirefoxAndroid = 'availability_firefox_android',
+  AvailabilitySafariIos = 'availability_safari_ios',
```

Comment:
For the availability columns to work, we need to add them to the openapi document
https://github.com/GoogleChrome/webstatus.dev/blob/b13707e03db34c79525b8f7a4ab350991bc52498/openapi/backend/openapi.yaml#L79-L87